04.ls/ls.rb

```diff
@@ -54,24 +54,25 @@ def show_long_format(filenames)
   total = file_details.map { |file_detail| file_detail[:file_block_size] }.sum / 2
   puts "total #{total}"
 
-  max_lengths = %i[nlink user_name group_name file_size].to_h do |key|
-    [key, find_max_length(key, file_details)]
-  end
+  max_lengths = %i[nlink user_name group_name file_size].to_h { |key| [key, find_max_length(key, file_details)] }
```

Comment:
行数を節約したい場合は、横に伸ばして解決するよりもメソッド化を検討したいですね。

```suggestion
  max_lengths = hogehoge
```